cranelift/codegen/src/machinst/mod.rs

```diff
@@ -340,6 +340,12 @@ pub struct MachCompileResult {
     pub value_labels_ranges: ValueLabelsRanges,
     /// Debug info: stackslots to stack pointer offsets.
     pub stackslot_offsets: PrimaryMap<StackSlot, u32>,
+    /// Basic-block layout info: block start offsets.
+    pub bb_starts: Vec<CodeOffset>,
```

Comment:
Can this be a `PrimaryMap` from `Block` to `CodeOffset`?

Reply:
Yes, that makes more sense, I agree; updated. Thanks!

Reply:
i see you said "updated" but it's still `Vec<CodeOffset>` in the PR ... missing commit somewhere?

Reply:
oh i sse the other resolved conversation about this, disregard!